common/lib/xblock/discussion/static/discussion_inline.js

```diff
@@ -0,0 +1,6 @@
+var $$course_id = "{{course_id}}";
```

Comment:
@pomegranited it is used in DiscussionUtil, or DiscussionModuleView - can't get exact reference off top of my head (but I'll try to find it). It won't work properly without it.